OrbitCore/OrbitModule.cpp

```diff
@@ -127,8 +127,8 @@ Function* Pdb::FunctionFromName(const std::string& a_Name) {
 }
 
 //-----------------------------------------------------------------------------
-Pdb::Pdb(uint64_t module_address, uint64_t load_bias,
-         const std::string& file_name, const std::string& module_file_name)
+Pdb::Pdb(uint64_t module_address, uint64_t load_bias, std::string file_name,
```

Comment:
What is the reason for passing this by value? I see that there is still only one copy because of std::move() below but the signature looks a bit off because of this.

Maybe replace with string_view (and drop std::move below)?

Reply:
It's the usual pass-by-value-and-move to support passing rvalues to constructors with no copies (but with moves) and no need for overloads. You can see that below file_name and module_file_name were already moved from, but that doesn't make sense if these two parameters are const references.